plugins/com.google.cloud.tools.eclipse.appengine.localserver/plugin.xml

```diff
@@ -100,4 +100,207 @@
             id="com.google.cloud.tools.eclipse.gcloud.runtime">
       </serverdefinition>
    </extension>
+   
+   <extension
+         point="org.eclipse.wst.server.core.runtimeTypes">
+      <runtimeType
+            class="com.google.cloud.tools.eclipse.appengine.localserver.runtime.CloudSdkRuntime"
+            description="%runtimeTypeDescription"
+            id="com.google.cloud.tools.eclipse.runtime.id1"
+            name="%runtimeTypeName"
+            vendor="%runtimeTypeVendor"
+            version="1">
+            <moduleType types="jst.web" versions="2.2, 2.3, 2.4, 2.5, 3.1"/>
+            <moduleType types="jst.connector" versions="1.0, 1.5, 1.6"/>
+            <moduleType types="jst.utility"  versions="1.0"/>
+      </runtimeType>
+   </extension>
+   <extension
+         point="org.eclipse.wst.server.ui.wizardFragments">
+      <fragment
+            class="com.google.cloud.tools.eclipse.appengine.localserver.ui.CloudSdkRuntimeWizardFragment"
+            id="com.google.cloud.tools.eclipse.runtime.fragment1"
+            typeIds="com.google.cloud.tools.eclipse.runtime.id1">
+      </fragment>
+   </extension>
+   <extension point="org.eclipse.wst.common.project.facet.core.facets">
+      <project-facet id="com.google.cloud.tools.eclipse.facet">
+        <label>%cloudsdkfacetlabel</label>
+        <description>%cloudsdkfacetdescription</description>
+      </project-facet>
+      <project-facet-version facet="com.google.cloud.tools.eclipse.facet" version="1">
+          <constraint>
+            <and>
+              <or>
+                  <requires facet="jst.web" version="2.2"/>
+                  <requires facet="jst.web" version="2.3"/>
+                  <requires facet="jst.web" version="2.4"/>
+                  <requires facet="jst.web" version="2.5"/>
+                  <requires facet="jst.web" version="3.0"/>
+                  <requires facet="jst.web" version="3.1"/>
+              </or>
+              <requires facet="java" version="[1.6"/>
```

Comment:
@elharo the syntax is correct:
https://eclipse.org/articles/Article-BuildingProjectFacets/tutorial.html#version.expressions
I've changed it to [1.7